### PR TITLE
Upgrade to latest Ember

### PR DESCRIPTION
**Pull request by devotox**

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
-      bower: {
-        dependencies: { }
-      }
-    },
-    {
-      name: 'ember-1.13',
+      name: 'ember-lts-2.8',
       bower: {
         dependencies: {
-          'ember': '~1.13.0'
```
Comment (mixonic) [inline config/ember-try.js]: This PR drops support for 1.13

File: ext/amp-index-template.html [hunk 2]
```diff
@@ -4,15 +4,16 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
 
-    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <!-- EMBER_CLI_FASTBOOT_HEAD -->
 
-    <script async src="https://cdn.ampproject.org/v0.js"></script>
-    <!-- EMBER_CLI_FASTBOOT_TITLE --><!-- EMBER_CLI_FASTBOOT_HEAD -->
```
Comment (mixonic) [inline ext/amp-index-template.html:10]: Is title now included in `HEAD`?

**Comment by mixonic** [comment]
@devotox thanks for working through this. Can you please squash this into a single commit? Are there any known issues you can identify? For example I saw the supported versions of Ember changed in CI. Please highlight anything significant.

File: ext/amp-index-template.html (hunk 2 above)
Comment (devotox) [reply ext/amp-index-template.html:10]: https://github.com/ember-fastboot/ember-cli-fastboot/blob/c83e885fb51c5b01f39e966ee54fb865d5bf41ca/index.js

Comment (devotox) [reply ext/amp-index-template.html:10]: Thats what it looks like there

File: config/ember-try.js (hunk 1 above)
Comment (devotox) [reply config/ember-try.js]: Yes i moved to the new version of ember try

**Pull request closed without merging**